docs/spark-procedures.md

```diff
@@ -277,6 +277,10 @@ Used to remove files which are not referenced in any metadata files of an Iceber
 | `dry_run`     |    | boolean   | When true, don't actually remove files (defaults to false) |
 | `max_concurrent_deletes` |    | int       | Size of the thread pool used for delete file actions (by default, no thread pool is used) |
 
+{{< hint warning >}}
+The timestamp within 24 hours cannot be set to `older_than`. For testing `remove_orphan_files`, configure `spark.testing` to true in the SparkSession object.
```

Comment:
I see value in explaining the 24 hour limit and even adding more context from the error message to the docs. However, I am not sure it is a good idea to mention our current internal hack for testing. I am concerned this would promote the misuse of the API. It also would prevent us from using another approach in the future.

Reply:
Thanks for the kind review and suggestion.

24 hours limitation would help users because I saw there are users who found this limitation after running `remove_orphan_files` with the interval within 24 hours. Let me add more context to this.

Regarding the `spark.testing` option, your concern makes sense, and I am also concerned the same.
The problem here I believe is that there are users who want to set `older_than` within 24 hours to run the `remove_orphan_files` tests or to check its behaviour. It's important to correctly guide Iceberg users for the`spark.testing` usage, at the same time, I think the `spark.testing` would help them. For example, it's possible to say:

*For testing purpose or checking the behaviour, not in production, you can set the interval within 24 hours by configuring `spark.testing` to true in SparkSession. If you set the interval within 24 hours, recommend setting `dry_run` option to true together.*

What do you think about my comment on adding `spark.testing` to the doc, and the phrase above?